client/ayon_maya/plugins/publish/validate_no_face_sets.py

```diff
@@ -0,0 +1,51 @@
+import ayon_maya.api.action
+import maya.cmds as cmds
+from ayon_core.pipeline.publish import (
+    OptionalPyblishPluginMixin,
+    PublishValidationError,
+    RepairAction,
+    ValidateContentsOrder,
+)
+from ayon_maya.api import plugin
+
+class ValidateNoFaceSets(plugin.MayaInstancePlugin,
+                         OptionalPyblishPluginMixin):
+    """Ensure the meshes don't have face sets """
```

Comment:
I have my doubts about the name and docstring. It seems to solely be about "Shader assignments per face" - not about "Face Sets" so I think we're better off wording this in a better way that it refers to shader assignments per face or alike.

Or at least to me it's confusing. I'd never heard of "Face Sets" in Maya before. :D

Reply:
It would be more concise to call "shaders per face". But the alembic options called "write face sets"